### ISSUE TITLE
Vieb 12.0.0 adblocker prevents some sites from loading

### ISSUE THREAD

**Issue by HashData** [Jelmerro/Vieb#563]
Some websites such as amazon.com and youtube.com don't load on Vieb 12.0.0 but they do on 11.0.0. This is the error message:

[522542:0521/161254.205774:ERROR:atom_cache.cc(229)] Add _NET_WM_STATE_MODAL to kAtomsToCache
MESA-INTEL: warning: Haswell Vulkan support is incomplete
libva error: /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so init failed
Error occurred in handler for 'GUEST_VIEW_MANAGER_CALL': Error: Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.
    at node:electron/js2c/renderer_init:2:16470
    at IpcRendererInternal.<anonymous> (node:electron/js2c/renderer_init:2:10723)
    at IpcRendererInternal.emit (node:events:518:28)
    at Object.onMessage (node:electron/js2c/renderer_init:2:8837)
Error occurred in handler for 'run-isolated-js-head-check': Error: Script failed to execute, this normally means an error was thrown. Check the renderer console for the error.
    at node:electron/js2c/renderer_init:2:16470
    at IpcRendererInternal.<anonymous> (node:electron/js2c/renderer_init:2:10723)
    at IpcRendererInternal.emit (node:events:518:28)
    at Object.onMessage (node:electron/js2c/renderer_init:2:8837)

**Specs**
 - OS: Ubuntu 
 - Version: 22.04.4 LTS

**Comment by Jelmerro** [comment]
This seems to be not our issue, as this is solved when turning off the adblocker. I can reproduce your stacktrace for different sites. Seems like the adblocker module broke something very badly by updating the `resources` file recently. For now you can temporarily disable the adblocker before visiting these sites. Please if you have to time try to reproduce the issue with [this project](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-electron-example) and then report the issue in that repo.

**Comment by se7en-x230** [comment]
Same problem now for FB. Disabling adblock works

**Comment by jkarstens** [comment]
Disabling adblocker does allow YouTube (et al) to load, but I think the issue might be in this repo.
In the devtools console on YouTube with adblocker enabled I see

```
Uncaught TypeError: Cannot read properties of null (reading 'innerText')
```

This console error goes away when adblocker is off (and the site loads normally)

Looks like this may be because `document.head` is `undefined` with the adblocker enabled which causes this line to error?
https://github.com/Jelmerro/Vieb/blob/419c43dbf84d7d9bb08ce68b958e66b6182c56c0/app/renderer/tabs.js#L589

YouTube with adblocker:
![image (23)](https://github.com/user-attachments/assets/00f89220-0b10-4df1-b7f8-f5bd0e7ce157)
![image (24)](https://github.com/user-attachments/assets/508d77cd-d494-4e1c-a4b4-11c87691f492)

YouTube without adblocker:

![image (25)](https://github.com/user-attachments/assets/4d854801-04a3-494e-8bea-99e60e260546)
![image (26)](https://github.com/user-attachments/assets/6468454f-bcfe-4949-af02-deca07c1e852)

**Comment by purgat0ri** [comment]
^ That would explain why one is still able to view embedded YouTube videos (e.g., https://www.reddit.com/r/planetemacs/comments/1eqqkhf/emacs_way_of_reading_novels_and_books_via_epub/) with adblockers enabled, even though the YouTube site does not load at all.

**Comment by Jelmerro** [comment]
I narrowed this down to the resources being loaded into the adblocker, not the entire adblocker, though once I remove them any video is blocked by requiring to "sign in to confirm i'm not a bot", so I'm not going to spend much time on this as it seems kind of pointless to use youtube directly like this, I'll continue to recommend invidious over plain youtube for this reason.

**Comment by Jelmerro** [comment]
should be fixed in 12.1.0 :tada:
